resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/KubernetesSuite.scala

```diff
@@ -265,7 +264,7 @@ class KubernetesSuite extends SparkFunSuite
       appLocator: String,
       isJVM: Boolean,
       pyFiles: Option[String] = None,
-      interval: Option[PatienceConfiguration.Interval] = None): Unit = {
+      executorPatience: Option[(Option[Interval], Option[Timeout])] = None): Unit = {
```

Comment:
a bit weird but stylecheck imposes a limit on the number of parameters which we reach here.